index.js

```diff
@@ -9,24 +9,21 @@ const mysql = require('mysql');
 const {
   backendPort,
   enableCORS,
-  mysqlConnParams,
-  privateKeyFile,
-  certificateFile,
-  caFile
+  mysqlConnParams
 } = require('./backend.config')
 
 // MySQL database connection
 const con = mysql.createConnection(mysqlConnParams);
 
 // Configure SSL certificate files
-const privateKey = fs.readFileSync(privateKeyFile, 'utf8');
-const certificate = fs.readFileSync(certificateFile, 'utf8');
-const ca = fs.readFileSync(caFile, 'utf8');
-const credentials = {
-	key: privateKey,
-	cert: certificate,
-	ca: ca
-};
+// const privateKey = fs.readFileSync(privateKeyFile, 'utf8');
```

Comment:
Did you left this on purpose or did you forget to remove it?

Reply:
Thanks!
It was a forgotten cleaning.